test/Common.h

```diff
@@ -98,6 +99,7 @@ struct CommonOptions
 
 private:
 	std::string evmVersionString;
+	std::string eofVersionString;
```

Comment:
```suggestion
	uint8_t eofVersion;
```

Reply:
Boost won't let me.

Reply:
Well, ok, I can not have it do it directly, but do it manually... why not, one second...

Reply:
Same here - it could use a comment about `uint8_t` vs `uint64_t`.

Reply:
I added comments to the parsing part of the option - do you think the member should *also* get *another* comment :-)?

Reply:
One comment is fine IMO.